runtime/.eslintrc.js

```diff
@@ -1,10 +1,8 @@
 module.exports = {
-    extends: [
-        'plugin:@typescript-eslint/recommended',
-        'prettier/@typescript-eslint',
```

Comment:
The Prettier configuration for `prettier/@typescript-eslint` no longer exists from what I can see with the updated plugins (see `package.json`).

OTOH I need to update the versions for the typescript-eslint plugins to be able to use `cli-style@7`.

Reply:
I think this comes from the `prettier` package...

Reply:
This should be added back in, probably, as it will fix the issues we added ignore comments for...?

Reply:
I'll give it another go with the Prettier package config.

Reply:
Nope.

```
varl@veronika:~/dev/dhis2/libs/app-runtime$ yarn lint
yarn run v1.22.4
$ d2-style js check && d2-style text check
d2-style > javascript

Oops! Something went wrong! :(

ESLint: 7.13.0

ESLint couldn't find the config "prettier/@typescript-eslint" to extend from. Please check that the name of the config is correct.

The config "prettier/@typescript-eslint" was referenced from the config file in "/home/varl/dev/dhis2/libs/app-runtime/runtime/.eslintrc.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Looks like the config was moved to a different package: `eslint-config-prettier`

Reply:
d56d132